Source/WebCore/dom/ScriptExecutionContext.h

```diff
@@ -142,7 +142,8 @@ class ScriptExecutionContext : public SecurityContext, public TimerAlignment {
 
     virtual void disableEval(const String& errorMessage) = 0;
     virtual void disableWebAssembly(const String& errorMessage) = 0;
-    virtual void setRequiresTrustedTypes(bool required) = 0;
+    virtual void setRequiresTrustedTypes(bool, bool) = 0;
```

Comment:
Same problem here, but it’s even worse because this one we can’t even understand in the header here. We need names for these two `bool` arguments, but really we need a different clearer idiom.